lighthouse-core/gather/driver.js

```diff
@@ -506,10 +543,17 @@ class Driver {
     const maxWaitMs = (options.flags && options.flags.maxWaitForLoad) ||
         Driver.MAX_WAIT_FOR_FULLY_LOADED;
 
-    return this.sendCommand('Page.enable')
+    const redirectTracking = {url};
```

Comment:
it's hard to tell what this is for and why it's passed into `_beginNetworkMontoring`'s `options` argument, maybe `redirectTrackingOptions/Container/Something` at least?